editor/store/selectors.js

```diff
@@ -300,6 +300,44 @@ export function isEditedPostEmpty( state ) {
 	);
 }
 
+/**
+ * Returns true if the post can be autosaved, or false otherwise.
+ *
+ * @param  {Object}  state Global application state
+ * @return {boolean}       Whether the post can be autosaved
+ */
+export function isPostAutosavable( state ) {
+	// If the post is autosaving, it is not autosavable.
+	if ( state.isAutosaving ) {
+		return false;
+	}
+
+	// If we don't already have an autosave, the post is autosavable.
+	if ( ! hasAutosave( state ) ) {
```

Comment:
Is a post autosavable if it has no content? Or are we relying on the consumer (e.g. `AutosaveMonitor`) to also check `isSavable` ? Should we want to impose that? If a post can't be autosaved without also being savable, should we just include that in `isPostAutosavable` ?

```js
if ( ! isPostSavable( state ) ) {
	return false;
}
```

Reply:
> Is a post autosavable if it has no content? 

Yes I think so, if thats a change (the post isn't new). For example, if you create a post with only a title and publish it, then edit the title and clear it. When autosave fires it will save the empty state as an autosave. I guess we could avoid this if thats desired, I can check the classic editor to see what the behavior is there for empty posts.

> If a post can't be autosaved without also being savable, should we just include that in isPostAutosavable ?

I couldn't find the `isPostSavable` or `isSavable` selectors, i think you mean `isEditedPostSaveable`

A post can't be saved if nothing has changed since the last save.
An autosave can be created - as long as the post has changed since the last autosave.